README.md

```diff
@@ -1,132 +1,106 @@
+![](src/public/assets/img/edge.png)
 
-The information of nodes and edges need to be stored in a *comma separated file* (also known as *csv* file); every single line of the file should follow this syntax:
-
-```
-property_1,property_2,...,property_n
-```
-
-where `property_x` is:
-- a `number`: an integer or a float (`.` must be used as decimal separator);
-- a `string`: they can be defined with or without `"` (or `'`) delimiters;
-- a `date` (or `datetime`);
-- a `boolean`;
-
-NOTE: regardless of the type, at the moment it's not possible to include a `,` in the value of a property.
-
-It's possible to include an *header* containing the names of the property. The header must be the first line of the file and it follow the same syntax as the property values.
-
-Example: *sample.csv*
-
-```
-first_name,last_name,age,country
-Roosvelt,Anderson,42,"US"
-Julianne,Redmond,35,"GB"
-Ginevra,Spacey,87,"AU"
-```
-
-## 2. Installation
-
-### 2.1 Setting the Plugin
-NOTE: for this part you need admin rights.
-
-1. Download the `importer.lke` plugin available [here](https://github.com/Linkurious/lke-plugin-csv-importer/releases/tag/v1.0.0).
-
-2. Copy the `.lke` archive in the folder `<lke-server>/data/plugins`
+Two sample CSV files, for nodes and edges, are available [here](https://github.com/Linkurious/lke-plugin-csv-importer/tree/master/sample%20csv):
+- `person.csv` contains information about person entities (a *node*);
+- `ping.csv` contains information about relationships between two person nodes (a *relationship*).
 
-3. On Linkurious Enterprise dashboard, go to `Admin` -> `Global configuration`
+## Set up the Plugin
 
-4. Scroll to the `Plugin settings` field
+> Make sure to have write access to the folder of Linkurious Enterprise as well as a Linkurious admin account.
 
-5. If there are no other plugins configured here, replace the whole content of the field with the following code and skip to step 7. \
-Otherwise, go to step 6
+1. Download the `csv-importer.lke` plugin available [here](https://github.com/Linkurious/lke-plugin-csv-importer/releases/tag/v1.0.0).
+2. Copy `csv-importer.lke` file to the following path `<linkurious>/data/plugins`
+3. Restart Linkurious Enteprise
+4. From the Linkurious Enterprise dashboard, go to `Admin` -> `Global configuration`
+5. Scroll to the `Plugin settings` field
+6. On the `Plugin settings` section, modify the content as follows:
 
     ```json
     {
-        "importer": {
-            "basePath": "importer"
-        }
-    }
-    ```
-
-6. On the `Plugin settings` field, add an new empty line before the last `}`, add in this line a comma (`,`) and, after the comma, paste the following code:
-
-    ```json
-    "importer": {
+      "importer": {
         "basePath": "importer"
+      }
     }
     ```
-
 7. Click on `Save`
 
-8. Done!
+## Access the plugin
```

Comment:
Having a CA is not a condition to access the plugin. should we just add the link here and as a more advanced option talk about CA?

Reply:
@aminerhanemi it kind of his. you need the source key in the URL. alternative is to create the URL yourself with the source key that seemed complicated to explain :D 
you are welcome to propose a sentence though